lib/sqlite.c

```diff
@@ -401,3 +440,160 @@ wtmpdb_read_all  (const char *db_path,
 
   return 0;
 }
+
+static int logrotate_callback(void *data, int argc, char **argv, char **azColName) {
+  (void)argc;
+  (void)azColName;
+  sqlite3 *db_dest = (sqlite3*)data;
+  char *error = NULL;
+  char *endptr;
+
+  const int type = atoi (argv[1]);
+  const char *user = argv[2];
+  const char *tty = argv[5]?argv[5]:"?";
+  const char *host = argv[6]?argv[6]:"";
+  const char *service = argv[7]?argv[7]:"";
```

Comment:
I use the defaults in src/wtmpdb.c because they are used as output on the console. In this case, we want to create a copy of the entry in a second database. Please don't overwrite the original entries, but use them, even if they are NULL.

Reply:
Yes !